exchange/adapter_builders.go

```diff
@@ -51,7 +51,7 @@ import (
 	"github.com/prebid/prebid-server/adapters/decenterads"
 	"github.com/prebid/prebid-server/adapters/deepintent"
 	"github.com/prebid/prebid-server/adapters/dmx"
-	"github.com/prebid/prebid-server/adapters/e_volution"
+	evolution "github.com/prebid/prebid-server/adapters/e_volution"
```

Comment:
Please remove other bidders aliases. This was probably automatically added by your IDE.

Reply:
in next commit i have solved:

The line has been removed